behave/helper/helper_web.py

```diff
@@ -10,6 +10,8 @@ def get_browser(browser):
             "download.default_directory": actual_path,
         }
         chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument("--start-maximized")
+        chrome_options.add_argument("--incognito")
```

Comment:
why incognito?

Reply:
Helpful when running tests locally as sometimes the previous browser session is used and can be logged in which causes issues for fox admin tests